pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>spring-boot-starter-parent</artifactId>
         <groupId>org.springframework.boot</groupId>
-        <version>2.7.18</version>
+        <version>3.0.0</version>
```

Comment:
Please update to the latest 3.0.x. as it should remove lots of vulnerabilities.

Reply:
driver is building fine only version  till 3.1.8, so should i uplift to  3.1.8  ?
In 3.2.x series getting errors.

Reply:
yes, you can uplift to 3.1.8. Make sure all tests are passed with this version. Thanks.

Reply:
done